WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Split Shipments/MoveToShipmentNotice.swift

```diff
@@ -0,0 +1,128 @@
+import SwiftUI
+
+struct MoveToShipmentNoticeViewModel {
+    enum MoveTo {
+        case existingShipment(index: Int)
+        case newShipment
+    }
+
+    let selectedItemsCount: Int
+    let existingShipmentsCount: Int
+    let currentShipmentIndex: Int?
+    let actionHandler: ((MoveTo) -> Void)
+}
+
+struct MoveToShipmentNotice: View {
+    let viewModel: MoveToShipmentNoticeViewModel
+
+    var body: some View {
+        HStack {
```

Comment:
This should be an adaptive stack instead to ensure the contents are readable when the font size is really large on a narrow device.